after/queries/go/injections.scm

```diff
@@ -1,137 +1,134 @@
-;; extends
+; extends
 
 ; inject sql into any const string with word query in the name
 ; e.g. const query = `SELECT * FROM users WHERE name = 'John'`;
 
-(
- (const_spec
-  name: (identifier) @_id
-  value: (expression_list (raw_string_literal) @injection.content))
-
- (#match? @_id ".*[Q|q]uery.*")
- (#set! injection.language "sql")
-)
-
-(short_var_declaration
-    left: (expression_list
-            (identifier) @_id (#match? @_id ".*[Q|q]uery.*"))
-    right: (expression_list
-             (raw_string_literal) @sql (#offset! @sql 0 1 0 -1))
-)
-
+((const_spec
+  name: (identifier) @_const
+  value: (expression_list 
+           (raw_string_literal) @sql))
+ (#any-of? @_const "query" "Query")
```

Comment:
The intention is to match any of 
userQuery = `select * from users where userID = ?`
query  = `select * fromxxx`
So maybe use either match or contains

Reply:
I feel that query could be used for other things though, this type of match could be wrongly applied leading to weird highlights. but up to you